sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

```diff
@@ -293,50 +291,174 @@
    },
    "outputs": [],
    "source": [
-    "MODEL_ID = \"bert-base-uncased\"\n",
+    "MODEL_ID = \"databricks/dolly-v2-12b\"\n",
     "TASK_NAME = \"fill-mask\""
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d7178a91",
+   "metadata": {},
+   "source": [
+    "#### Check if the model already exists in AzureML registries"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "90df6db4",
+   "id": "944ea15c",
    "metadata": {},
    "outputs": [],
    "source": [
-    "COMPUTE = compute_name"
+    "from typing import Optional\n",
+    "from azure.ai.ml.entities import Model\n",
+    "\n",
+    "\n",
+    "def does_model_exist_in_registry(\n",
+    "    credential: DefaultAzureCredential, registry_name: str, model_id: str\n",
+    ") -> Optional[Model]:\n",
+    "    \"\"\"\n",
+    "    Checks if the model exists in the registry.\n",
+    "\n",
+    "    :param credential: credential to access the registry.\n",
+    "    :param registry_name: name of the registry.\n",
+    "    :param model_id: model id.\n",
+    "    :return: Model object.\n",
+    "    \"\"\"\n",
+    "    try:\n",
+    "        ml_client = MLClient(credential=credential, registry_name=registry_name)\n",
```

Comment:
nit: latest SDK version supports use of `label="latest"`. Should we use that instead?

Reply:
Done.